File: 217-contains-duplicate/contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        hm = {}
        for i in nums:
            if i not in hm:
                hm[i] = 0
            hm[i] += 1
        print(hm.values())

        for i in hm.values():
            if i!=1:
                return True
        return False

            

File: 217-contains-duplicate/contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashMap<Integer, Integer> hm = new HashMap<>();
        for(int i : nums){
            if(!(hm.containsKey(i))){
                hm.put(i,0);
            }
            int val = hm.get(i);
            hm.put(i, ++val);
        }
        for(int i: nums){
            if(hm.get(i)>1){
                return true;
            }
        }
        return false;
    }
}